docs/src/main/asciidoc/_admin-guide/subsystem-configuration/Elytron_OIDC_Client.adoc

```diff
@@ -141,6 +143,26 @@ configuration). If the subsystem detects that the `OIDC` mechanism is indeed req
 activate the authentication mechanism automatically. Otherwise, no activation will occur and deployment
 will continue normally.
 
+=== Scope Configuration
+
+OpenID Connect allows the client to use scope values to request access to specific privileges for access tokens. An access token's scopes determine what resources will be available when using the access token to access an OAuth2 protected endpoint. The OpenID Connect https://openid.net/specs/openid-connect-core-1_0.html#ScopeClaims[specification] defines a few scope values including:
+
+* `profile` : requests access to the End-User's default profile Claims, such as name, family_name, given_name, middle_name, nickname, preferred_username, profile, picture, website, gender, birthday etc. 
+
+* `email` : requests access to the email and email_verified Claims.
+
+* `address` :  requests access to the address Claim. 
+
+* `phone` : requests access to the phone_number and phone_number_verified Claims.
+
+=== Subsystem Configuration
+It is possible to configure the scope values that would be used in the OIDC authentication request using the `scope` attribute of a `secure-deployment` or a `secure-server` resource in the `elytron-oidc-client` subsystem. The scope values must be added as a space separated list of lowercase ascii values.
+
+=== Deployment Configuration
+It is also possible to configure the scope values that would be used via the scope attribute in a deployment's `oidc.json` file. The value for scope must be a space separated list of strings.
+Scopes can be used to request that specific sets of information be made available as Claim Values.
+Note: If both the deployment and subsystem configurations are configured, then the subsystem configuration will override the deployment values.
```

Comment:
s/configurations are configured/configuration is present

Reply:
fixed! 
Thank you for the comments